### PR TITLE
New news items and  revised about page from Sandy

### PR DESCRIPTION
**Pull request by jpstroop**

### CODE REVIEW

**Comment by jpstroop** [comment]
There's some CSS tweaking in here. I think I found the best spot. @axamei may have comments?

File: app/assets/stylesheets/components/_section.scss [hunk 1]
```diff
@@ -8,3 +8,15 @@ section {
   padding: 1rem;
   width: 100%;
 }
+
+.news {
```
Comment (axamei) [inline app/assets/stylesheets/components/_section.scss]: I would create a new partial for news (_news.scss) and include in application.scss

File: app/assets/stylesheets/components/_section.scss [hunk 2]
```diff
@@ -8,3 +8,15 @@ section {
...
+  h2 {
+    text-align: left;
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+  h2+p {
+    margin-top: 0;
+  }
+
+}
```
Comment (axamei) [inline app/assets/stylesheets/components/_section.scss]: Add new rule for  .news-entry to have `padding: 1rem;`

File: app/views/pages/news.html.erb [hunk 3]
```diff
@@ -1,5 +1,17 @@
 <section id="news">
   <div class="wrap">
     <h1>Latest News</h1>
+    <section class="news">
```
Comment (axamei) [inline app/views/pages/news.html.erb]: Remove section tags for individual news entries

File: app/views/pages/news.html.erb [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+      <div class="wrap">
```
Comment (axamei) [inline app/views/pages/news.html.erb]: Change `wrap` to `news-entry`

File: app/views/pages/news.html.erb [hunk 5]
```diff
@@ -1,5 +1,17 @@
...
+        <h2>Cicognara at DLF</h2>
+        <p>Princeton's Sandra Brooke (Marquand Librarian) and Jon Stroop (Library Applications Development Manager) will be discussing the Digital Cicognara Library on November 7, 2016, in a IIIF-focused session at the <a href="https://www.diglib.org/forums/2016forum/">Digital Library Federation Forum</a> in Milwaukee, Wisconsin. <a href="https://dlfforum2016.sched.org/event/8LGE/m3b-iiif">"Uncommon Objects of Delight: Recreating Count Leopoldo Cicognara's Library"</a> will address the DCL’s academic goals and technical implementation and show how the project is leveraging Blacklight and the International Interoperability Framework technologies.</p>
+      </div>
+    </section>
+    <section class="news">
```
Comment (axamei) [inline app/views/pages/news.html.erb]: Remove section tag for individual news entries.

File: app/views/pages/news.html.erb [hunk 6]
```diff
@@ -1,5 +1,17 @@
...
+      <div class="wrap">
```
Comment (axamei) [inline app/views/pages/news.html.erb]: Change `wrap` to `news-entry`

**Comment by jpstroop** [comment]
OK, changes made--it wasn't exactly what you suggested, but I think I have the gist. Here's what it looks like:

image

What do you think of the dates?

**Comment by axamei** [comment]
Let's move the dates from the h2. Put it on its own line either below the news title or below the news paragraph.

**Comment by jpstroop** [comment]
I kind of like them on the same line--is it the look of it that you don't like, or the semantics of having them in `h2`?

**Comment by axamei** [comment]
Both. Do you expect short titles all the time? If so, you can keep it on the same line and float it to the right for better visual separation.

**Comment by jpstroop** [comment]
The floating looked weird in wider windows. Here's what's coming:
image

**Comment by axamei** [comment]
👍

**Pull request merged**